testcode/lib/conio.c

```diff
@@ -14,6 +14,15 @@
 #include <string.h>
 #include <stdlib.h>
 
+#if defined(__GAMATE__)
+/* there is not enough screen space to show all 256 characters at the bottom */
```

Comment:
I see - so this isn't a limitation of the Gamate CONIO implementation (supporting only chars  < 128) but rather specific to this test program.